app/assets/javascripts/admin/addon/components/admin-config-areas/color-palette.gjs

```diff
@@ -250,63 +255,88 @@ export default class AdminConfigAreasColorPalette extends Component {
   }
 
   <template>
-    <Form
-      data-palette-id={{@colorPalette.id}}
-      @data={{this.data}}
-      @onSubmit={{this.handleSubmit}}
-      @onRegisterApi={{this.onRegisterApi}}
-      as |form transientData|
-    >
-      <div>
-        <div class="admin-config-color-palettes__top-controls">
-          {{#if this.editingName}}
-            <form.Field
-              @name="name"
-              @showTitle={{false}}
-              @title={{i18n "admin.config_areas.color_palettes.palette_name"}}
-              @validation="required"
-              @format="full"
-              @onSet={{this.handleNameChange}}
-              as |field|
-            >
+    <div class="admin-config-area__primary-content">
+      <div class="admin-config-color-palettes__back">
+        <LinkTo @route="adminConfig.colorPalettes">
+          {{icon "angle-left"}}
+          {{i18n "admin.customize.colors.back_to_colors"}}
+        </LinkTo>
+      </div>
+      <Form
+        data-palette-id={{@colorPalette.id}}
+        @data={{this.data}}
+        @onSubmit={{this.handleSubmit}}
+        @onRegisterApi={{this.onRegisterApi}}
+        as |form transientData|
+      >
+        <div class="admin-config-color-palettes__header">
+          <div class="admin-config-color-palettes__top-controls">
+            {{#if this.editingName}}
+              <form.Field
+                @name="name"
+                @showTitle={{false}}
+                @title={{i18n "admin.config_areas.color_palettes.palette_name"}}
+                @validation="required"
```

Comment:
When validation kicks, it looks weird 

<img width="729" height="215" alt="Image" src="https://github.com/user-attachments/assets/3db7d938-fa2e-4a4e-9c64-ce370f9dd104" />